tests/foreman/longrun/test_oscap.py

```diff
@@ -150,290 +136,9 @@ def update_scap_content(module_org):
-            'organization': module_org.name,
-        }
-    )
-    result = TailoringFiles.info({'name': tailoring_file_name})
-    assert result['name'] == tailoring_file_name
-    # Creates oscap_policy.
-    scap_id, scap_profile_id = fetch_scap_and_profile_id(
-        OSCAP_DEFAULT_CONTENT['rhel7_content'], OSCAP_PROFILE['security7']
-    )
-    make_scap_policy(
-        {
-            'scap-content-id': scap_id,
-            'hostgroups': hgrp_name,
-            'deploy-by': 'puppet',
-            'name': policy_name,
-            'period': OSCAP_PERIOD['weekly'].lower(),
-            'scap-content-profile-id': scap_profile_id,
-            'weekday': OSCAP_WEEKDAY['friday'].lower(),
-            'tailoring-file-id': tailor_result['id'],
-            'tailoring-file-profile-id': tailor_result['tailoring-file-profiles'][0]['id'],
-            'organizations': module_org.name,
-        }
-    )
-    # Creates vm's and runs openscap scan and uploads report to satellite6.
-    with VMBroker(
-        nick=DISTRO_RHEL7,
-        host_classes={'host': ContentHost},
-        target_cores=OSCAP_TARGET_CORES,
-        target_memory=OSCAP_TARGET_MEMORY,
-    ) as vm:
-        host_name, _, host_domain = vm.hostname.partition('.')
-        vm.install_katello_ca(default_sat)
-        vm.register_contenthost(module_org.name, ak_name[DISTRO_RHEL7])
-        assert vm.subscribed
-        Host.update(
-            {
-                'name': vm.hostname.lower(),
-                'lifecycle-environment': lifecycle_env.name,
-                'content-view': content_view.name,
-                'hostgroup': hgrp_name,
-                'openscap-proxy-id': default_proxy,
-                'organization': module_org.name,
-                'puppet-environment-id': puppet_env.id,
-            }
-        )
-        Host.set_parameter(
-            {
-                'host': vm.hostname.lower(),
-                'name': 'remote_execution_connect_by_ip',
-                'value': 'True',
-                'parameter-type': 'boolean',
-            }
-        )
-        vm.configure_puppet(
-            rhel_repo={'rhel7': settings.repos.rhel7_os}, proxy_hostname=default_sat.hostname
-        )
-        result = vm.run('cat /etc/foreman_scap_client/config.yaml | grep profile')
-        assert result.status == 0
-        # Runs the actual oscap scan on the vm/clients and
-        # uploads report to Internal Capsule.
-        vm.execute_foreman_scap_client()
-        # Assert whether oscap reports are uploaded to
-        # Satellite6.
-        arf_report = Arfreport.list({'search': f'host={vm.hostname.lower()}', 'per-page': 1})
-        assert arf_report is not None
-        Arfreport.delete({'id': arf_report[0].get('id')})
-
-
-@pytest.mark.upgrade
-@pytest.mark.tier4
-@pytest.mark.parametrize('distro', [DISTRO_RHEL7, DISTRO_RHEL8])
+@pytest.mark.parametrize('distro', [DISTRO_RHEL6, DISTRO_RHEL7, DISTRO_RHEL8])
```

Comment:
Any reasons to add RHEL6 since it was not earlier ? I would instead go for RHEL9. 

Just a thought, not a blocker comment !